src/tensorlake/functions_sdk/functions.py

```diff
@@ -296,6 +270,8 @@ def get_output_model(self) -> Any:
         extract_method = self.indexify_function.run
         type_hints = get_type_hints(extract_method)
         return_type = type_hints.get("return", Any)
+        if get_origin(return_type) is RouteTo:
```

Comment:
Interesting. This is not related to this PR, but do I understand correctly that due to the use of `get_origin` we're strongly relying on type hints provided by users for SDK to work correctly?

If yes then this looks not quite correct to me because Python is a dynamic language and it looks like we can use actual runtime types here without any cons. Does this sound right or not?

Reply:
This is being used by graph validation; I don't think we can use actual runtime types here.

Reply:
Hmmmm if graph validation fails then it prevents the graph from getting deployed. Might not be what we want (if users don't want to do type annotations), just fyi.